docs/dev-notes/DEVELOPMENT.md

```diff
@@ -0,0 +1,135 @@
+- Setup react native development environment on your machine by following commands below
+
+> This guide assumes you have [homebrew](https://brew.sh/) installed and combines most of the commands you would need to setup the environment on your machine. If you want to read full guide, visit the [official docs](https://reactnative.dev/docs/environment-setup) -> React Native CLI Quickstart -> Your Development and Target OS
+
+  - Install node using `brew install node` (Node 14 or newer required, prefer LTS versions)
+  - Check if you have ruby installed on your machine by running `ruby --version` (React Native uses [this version](/.ruby-version) of Ruby)
+
+> Currently, macOS 12.5.1 is shipped with Ruby 2.6.8, which is not what is required by React Native. We recommend installing some Ruby Manager to have different versions of Ruby installed on your machine without breaking other projects. [React Native docs have listed a few](https://reactnative.dev/docs/environment-setup#ruby), you can choose any of them and skip instructions for Ruby Version Manager if you want to go with a different option.
+
+ - If you do not have required version of ruby installed
+   - Install [rbenv](https://github.com/rbenv/rbenv) for managing Ruby versions by running `brew install rbenv ruby-build`
+   - Load rbenv in your shell by running `rbenv init` and restarting the shell
+   - Make sure you have the required ruby version set locally. You can do this by running `rbenv local 2.7.6`
+ - Make sure you have Xcode Command Line Tools installed
+
+> **Warning**
+> 
+> **Please make sure you are at root directory of the project before running commands mentioned below**
+
+### Connect the app to a workspace in Customer.io
+
+- Make a copy of `env.sample.js` by running `cp env.sample.js env.js` at project root directory
+- Update `env.js` with your `siteId` and `apiKey`
+
+### Install Dependencies
+
+Before running any app, make sure the dependencies are installed by running following command:
+
+```
+yarn install
+```
+
+In case you add/update dependencies later, you will need to run this command again to install the updated dependencies.
+
+### Android
+
+*Note: It's assumed that you have Android Studio installed on your machine before continuing*
+
+This project requires Firebase setup in order to build the app. You can quickly get the app to build with following command
+
+```
+cp android/app/google-services.json.example android/app/google-services.json
+```
+
+But if you want to do anything like push notifications, you will want to follow [Firebase setup](https://firebase.google.com/docs/android/setup) to add a new Android app to your Firebase project. This will give you a `google-services.json` file to download. Save this file at `android/app/google-services`. 
+
+#### Option 1: Using Command Line
```

Comment:
Option 1 for what? I don't see a header defining what we need options for.

Reply:
Oh, I see now. It's how to compile the app for iOS. 

```suggestion
#### Build app option 1: Using Command Line
```

Would this be more helpful of a header?

Reply:
Yeah, descriptive names are always helpful. I'll match other headers with this too.